rust/agama-lib/share/profile.schema.json

```
@@ -336,6 +336,29 @@
       "type": "object",
       "additionalProperties": false,
       "properties": {
+        "generalState": {
+          "title": "Network general state settings",
+          "type": "object",
+          "properties": {
+            "connectivity": {
+              "title": "Determines whether the user is able to access the Internet",
```

```suggestion
              "title": "Whether the user is able to access the Internet",
```

Just for consistency.